### PR TITLE
[558001] Add Copy and Paste as Description Link functionality

### PR DESCRIPTION
**Pull request by sandupostaru**
- Add Copy and Paste as description link option

- Add documentation
-- Complete the Description Section
-- Add Tips and Tricks Section

- Add JUnit

Bug: 558001
Change-Id: I62496ae1935c1475609d46e47e39361d3904452a
Signed-off-by: Sandu Postaru <sandu.postaru@thalesgroup.com>

### CODE REVIEW

File: core/plugins/org.polarsys.capella.core.ui.properties.richtext/plugin.xml [hunk 1]
```diff
@@ -82,4 +82,80 @@
             class="org.polarsys.capella.core.ui.properties.richtext.CapellaMDERichtextWidgetEditorImpl">
       </editorWidgetContribution>
    </extension>
+   <extension
+         point="org.eclipse.ui.commands">
+      <command
+            categoryId="org.polarsys.capella.core.platform.sirius.ui.navigator"
+            defaultHandler="org.polarsys.capella.core.ui.properties.richtext.handlers.CopyAsDescriptionLinkHandler"
+            id="org.polarsys.capella.core.ui.properties.richtext.CopyAsDescriptionLink"
+            name="%command.copyAsDescriptionLink.name ">
```
Comment (pdulth) [inline core/plugins/org.polarsys.capella.core.ui.properties.richtext/plugin.xml]: Is it working with a space at the end?

File: tests/plugins/org.polarsys.capella.test.richtext.ju/src/org/polarsys/capella/test/richtext/ju/testcases/RichTextLinksClipboardTest.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
+package org.polarsys.capella.test.richtext.ju.testcases;
```
Comment (pdulth) [inline tests/plugins/org.polarsys.capella.test.richtext.ju/src/org/polarsys/capella/test/richtext/ju/testcases/RichTextLinksClipboardTest.java]: Copyright

File: core/plugins/org.polarsys.capella.core.ui.properties.richtext/plugin.xml (hunk 1 above)
Comment (sandupostaru) [reply core/plugins/org.polarsys.capella.core.ui.properties.richtext/plugin.xml]: Yes it works, but I will remove the space.

File: tests/plugins/org.polarsys.capella.test.richtext.ju/src/org/polarsys/capella/test/richtext/ju/testcases/RichTextLinksClipboardTest.java (hunk 2 above)
Comment (sandupostaru) [reply tests/plugins/org.polarsys.capella.test.richtext.ju/src/org/polarsys/capella/test/richtext/ju/testcases/RichTextLinksClipboardTest.java]: Done

**Comment by sandupostaru** [comment]
Tests are ok: https://ci.eclipse.org/capella/job/capella-product/view/change-requests/job/PR-20/3/

**Pull request merged**